### PR TITLE
Footer PRA-125 v 2

### PR DESCRIPTION
**Pull request by ElenaGor8**
## One Line Description
Changed footer, addressed comments in the previous PR, but by mistake created a different branch.
old pr: https://github.com/Code-the-Dream-School/dd-prac-team1-front/pull/30

## Requirements
provide footer throughout the app

_What this code should do_

## Notes

_Any additional info_

## Test Steps

_Instructions on how to test the changes_

## UI/UX Outcome

_Attach a screenshot here_
footer

## Checklist

- The Linear ID is in the PR title
- My code follows best practices
- Documentation is included where needed

### CODE REVIEW

**Comment by ElenaGor8** [comment]
@rebekahcallkacz
I made changes to the Footer, will you please review the recent push?
I have two questions:

the footer doesn't show as intended as it covers the "create account" button on homepage.
Do we need to change Box height settings in index.tsx?
I see that the footer background color got picked up from CustomTheme as brandGray, but the green color in the hover borderBottom: "2px solid green" is not our green. What could be the reason for that?

**Review by AnnaPestova1 — APPROVED** [review]

**Comment by rebekahcallkacz** [comment]
> @rebekahcallkacz I made changes to the Footer, will you please review the recent push? I have two questions:
> 
> the footer doesn't show as intended as it covers the "create account" button on homepage. Do we need to change Box height settings in index.tsx? I see that the footer background color got picked up from CustomTheme as brandGray, but the green color in the hover borderBottom: "2px solid green" is not our green. What could be the reason for that?

You have the footer set up to be at the bottom of the viewport which is not ideal when many of your pages will be longer than that. I would just have the footer be at the bottom of the content, not the bottom of the viewport and I would modify the homepage so that it doesn't take up 100% of the viewport. I'll note in my code review what lines I would change.

File: src/components/Footer.tsx
Comment (rebekahcallkacz) [inline src/components/Footer.tsx:9]: Delete this line

File: src/components/Footer.tsx
Comment (rebekahcallkacz) [inline src/components/Footer.tsx:10]: Delete this line

File: src/components/Footer.tsx
Comment (rebekahcallkacz) [inline src/components/Footer.tsx:24]: I'm not sure why, but Chakra doesn't seem to be able to pass colors using this format. You need to do this instead:
```
              <Text
                _hover={{
                  borderBottomColor: "green",
                  borderBottomWidth: "2px",
                  borderBottomStyle: "solid"
                }}>
```

**Review by rebekahcallkacz — APPROVED** [review]
See my other comments for how to fix your styling issues. I'm approving now with the assumption that you will make those changes before merging this PR (I don't want you to have to wait on me for an approval when it's just a few styling lines to change).

**Pull request merged**
